tests/mesolve/test_euler.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import qutip as qt
+
+import torchqdynamics as tq
+
+
+def test_mesolve_euler():
+    """Cheap test of the Euler method of mesolve."""
+    # parameters
+    n = 8
+    kappa = 1.0
+    delta = 2 * np.pi
+    alpha0 = 1.0
+
+    # operators
+    a = qt.destroy(n)
+    adag = a.dag()
+    H = delta * adag * a
+    jump_ops = [np.sqrt(kappa) * a, np.sqrt(kappa) * a]
```

Comment:
Maybe better to use two different jump operators? You could use the identity for the second jump operator actually, just for sanity check that it doesn't do anything. And this way you still check the `jump_ops` batching.